src/main/java/com/ldtteam/structurize/util/RenderUtils.java

```diff
@@ -175,7 +175,7 @@ public RenderTypes(final String nameIn,
             RenderType.State.builder()
                 .setLineState(new RenderState.LineState(OptionalDouble.empty()))
                 .setLayeringState(VIEW_OFFSET_Z_LAYERING)
-                .setTransparencyState(GLINT_TRANSPARENCY)
+                .setTransparencyState(NO_TRANSPARENCY)
```

Comment:
`optifine compat ? no_trans : glint_trans`
I remember this being problem with some shaders, I also remember creating a fix for that - not sure where it went tho :D
Root cause is probably this being rendered in wrong framebuffer (which you really don't need to care about, will fix that once forge renderWorldEvents get redesigned)

Reply:
but do we need the glint transparency in the first place here?

Reply:
I chose this because entity outlining is also glint, so the player gets that it's just outline and not something special